samples/nrf9160/serial_lte_modem/src/slm_at_gps.c

```diff
@@ -61,6 +49,7 @@ static u64_t ttft_start;
 
 /* global variable defined in different files */
 extern struct at_param_list m_param_list;
+extern char m_buf[64];
```

Comment:
Is this array a bit small? NMEA strings are max. 84 bytes as an example.

Reply:
Sorry, typo. The buffer is reserved in main.c, which is buf[128]. Thanks!

Reply:
I'll make the length configurable